jacoco-maven-plugin/src/org/jacoco/maven/ReportAggregateMojo.java

```diff
@@ -110,6 +110,22 @@ void loadExecutionData(final ReportSupport support) throws IOException {
 		}
 	}
 
+	private int executionDataFilesCount() {
+		final FileFilter filter = new FileFilter(dataFileIncludes,
+				dataFileExcludes);
+		try {
+			int count = 0;
+			for (final MavenProject dependency : findDependencies(
```

Comment:
Yes,  understood. We discussed and the report-aggregate goal is not currently intended to include the reported upon project. #414 appears to reverse this.  Should #414 be  merged this will need to change.